### PR TITLE
[NoJira] improve consistency with the monorepo

### PR DESCRIPTION
**Pull request by matteo-hertel**
Just some consistency improvement with the monorepo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Backpack
```
Comment (matteo-hertel) [inline README.md]: I'd like this to be `Backpack Design System` like the monorepo, thoughts?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Unreleased
+# Backpack changelog
```
Comment (matthewdavidson) [inline CHANGELOG.md]: Should be `Backpack iOS changelog`

File: README.md (hunk 1 above)
Comment (matthewdavidson) [reply README.md]: How about `Backpack iOS`?

File: decisions/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Backpack decisions
```
Comment (matthewdavidson) [inline decisions/README.md]: Hmm I would have thought we should add this when we actually have decisions ...

File: UNRELEASED.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Unreleased
```
Comment (k0nserv) [inline UNRELEASED.md]: If we are going to add this file we need to update the instructions in `CONTRIBUTING.md` for releases. Also do we really need this in this repo? The reason we have it in the main repo was the volume of change we accumulate?

**Comment by matteo-hertel** [comment]
The key for this PR is to have as much consistency as possible across our repos, so switching between them is less painless, unless major concerns I'd really like this changes to go through 

As per the name I believe we had that discussion back in the days and we chose not to add iOS as it's in the name of the repo (or something like that)

File: README.md (hunk 1 above)
Comment (shaundon) [reply README.md]: I'm happy with that

**Comment by k0nserv** [comment]
I don't feel as strongly about consistency, but I am happy with the change 🤷‍♂️

**Comment by matthewdavidson** [comment]
I agree with consistency, but the decisions in particular seem to be YAGNI. Also the names surely need to be different ... just look at the URL ^^

**Comment by matteo-hertel** [comment]
I can rename to Backpack iOS if we're all happy with it cc @shaundon @georgegillams 

As per the decisions, I don't see why YAGNI, it'll drive consistency in the repos and it'll save somebody else the copy paste from the Backpack repo when we come down to make a decision, does it look weird having it? if so I'll happily remove it

**Comment by georgegillams** [comment]
I like keeping things as consistent as possible - I imagine as time goes on we will have some members of the squad who predominantly work on one platform over another. We're gonna get stupid errors if the process varies between every repo

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ Tests can be ran as usual from Xcode(Product -> Test or cmd+U). Snapshot tests s
 
 > Backpack team only
 
-To issue a new release make sure you've set the project up as above, that you have push acess to the Backpack CocoaPod and that you're logged in to CocoaPods trunk. `bundle exec pod trunk me` should print your info and including the Backpack pod in the output. Make sure you've added a new entry in the `CHANGELOG.md` with the version you're publishing then run `bundle exec rake release`.
+To issue a new release make sure you've set the project up as above, that you have push acess to the Backpack CocoaPod and that you're logged in to CocoaPods trunk. `bundle exec pod trunk me` should print your info and including the Backpack pod in the output. Make sure you've added a new entry in the `UNRELEASED.md` and update the `CHANGELOG.md` file with the version you're publishing then run `bundle exec rake release`.
```
Comment (k0nserv) [inline CONTRIBUTING.md]: This is not correct, when releasing you should move the entries from `UNRELEASED.md` to `CHANGELOG.md` and add them under the header with the new version and then run `bundle exec rake release`

**Pull request merged**